src/components/general-table/BulkSelect.js

```diff
@@ -16,6 +16,7 @@ const BulkSelect = ({
   displayedRowsLength,
 }) => {
   const isAllSelected = checkedRows.length === displayedRowsLength;
+  const isPartiallySelected = checkedRows.length > 0 ? null : false;
```

Comment:
Why it should be null?

Reply:
To enable the partial checkbox icon the `-` when selecting one or more (not all) it looks for null not false as it was previously getting. 

Here is the documentation for checkbox. https://www.patternfly.org/v4/components/checkbox#checkbox